src/actors/context-menus/rune-context-menu.ts

```diff
@@ -70,22 +72,14 @@ export const runeMenuOptions = (
     name: localize("RQG.ContextMenu.ViewDescription"),
     icon: ContextMenuRunes.ViewDescription,
     condition: (el: JQuery) => {
-      const itemId = getRequiredDomDataset(el, "item-id");
-      let firstItemEl = el[0];
-      while ((firstItemEl?.previousElementSibling as HTMLElement)?.dataset?.itemId === itemId) {
-        firstItemEl = firstItemEl.previousElementSibling as HTMLElement;
-      }
-      return !!getDomDataset($(firstItemEl), "journal-id");
+      const rqid = findDatasetValueInSelfOrAncestors(el[0] as HTMLElement, "runeRqid");
+      return rqid ? true : false;
```

Comment:
I think you could simplify this to `return !!findDatasetValueInSelfOrAncestors(el[0] as HTMLElement, "runeRqid");`

Reply:
See above.